### DISCUSSION TITLE
有關於畫面的問題想要詢問

### DISCUSSION THREAD

**Discussion by robinsonlin603** [Glossika-PL/stop-watch-widget#1] [Q&A] [answered]
1. 中間的 白、灰 兩個圓圈是否需要隨著切換更換不同顏色
![image](https://github.com/Glossika-PL/stop-watch-widget/assets/93252094/d1199363-b859-4079-9e33-d6762acebfa9)

**Comment by Glossika-PL** [comment, marked as answer]
Hi robinsonlin603,
請問是想確認這兩顆嗎圓圈嗎？
<img width="129" alt="image" src="https://github.com/Glossika-PL/stop-watch-widget/assets/109053974/a9ea691c-d483-4965-bdc0-c276c83280a7">

那張圖是示意圖，不需要呈現中間白灰圓圈，實際可以參考 demo 影片
感謝提出困惑，我會再修改示意圖

**Reply by robinsonlin603** [reply]
感謝您
